frontend/TestRun/ResourcesInfo.svelte

```diff
@@ -88,6 +89,7 @@
                 navigator.clipboard.writeText(
                     `${regions} hydra clean-resources --backend ${backend} --test-id ${run_id}`
                 );
+                sendMessage("success", `\`${regions} hydra clean-resources --backend ${backend} --test-id ${run_id}\` has been copied to your clipboard`);
```

Comment:
can we put the text into variable, and not create it twice ? it's a recipe of getting it out of sync, and would be a bit confusing.